### PR TITLE
completed module_6

### PR DESCRIPTION
**Pull request by vladSHOKO**

### CODE REVIEW

File: module_6/library.php [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+<h1 class="<?=$red?>"><?= $title ?></h1>
+<div>Авторов на портале: <?= $countOfAuthors ?></div>
+
+<p class="grey">Книга <b><?=$library['books'][0]['title']?></b>, ее написал <?=$library['authors']['john_makkormik@example.com']['name']?> <?=$library['authors']['john_makkormik@example.com']['birthYear']?> (<a href="#"><?=$library['books'][0]['author']?></a>)</p>
+<p>Книга <b><?=$library['books'][1]['title']?></b>, ее написал <?=$library['authors']['martin_robert@example.com']['name']?> <?=$library['authors']['martin_robert@example.com']['birthYear']?> (<a href="#"><?=$library['books'][1]['author']?></a>)</p>
+<p class="grey">Книга <b><?=$library['books'][2]['title']?></b>, ее написал <?=$library['authors']['martin_robert@example.com']['name']?> <?=$library['authors']['martin_robert@example.com']['birthYear']?> (<a href="#"><?=$library['books'][2]['author']?></a>)</p>
+<p>Книга <b><?=$library['books'][3]['title']?></b>, ее написал <?=$library['authors']['martin_fauler@example.com']['name']?> <?=$library['authors']['martin_fauler@example.com']['birthYear']?> (<a href="#"><?=$library['books'][3]['author']?></a>)</p>
+
+</body>
+</html>
```
Comment (Lehersun) [inline module_6/library.php:73]: observe the code style please

File: module_6/library.php [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+</head>
+<body class="main">
+<h1 class="<?=$red?>"><?= $title ?></h1>
+<div>Авторов на портале: <?= $countOfAuthors ?></div>
+
+<p class="grey">Книга <b><?=$library['books'][0]['title']?></b>, ее написал <?=$library['authors']['john_makkormik@example.com']['name']?> <?=$library['authors']['john_makkormik@example.com']['birthYear']?> (<a href="#"><?=$library['books'][0]['author']?></a>)</p>
+<p>Книга <b><?=$library['books'][1]['title']?></b>, ее написал <?=$library['authors']['martin_robert@example.com']['name']?> <?=$library['authors']['martin_robert@example.com']['birthYear']?> (<a href="#"><?=$library['books'][1]['author']?></a>)</p>
+<p class="grey">Книга <b><?=$library['books'][2]['title']?></b>, ее написал <?=$library['authors']['martin_robert@example.com']['name']?> <?=$library['authors']['martin_robert@example.com']['birthYear']?> (<a href="#"><?=$library['books'][2]['author']?></a>)</p>
+<p>Книга <b><?=$library['books'][3]['title']?></b>, ее написал <?=$library['authors']['martin_fauler@example.com']['name']?> <?=$library['authors']['martin_fauler@example.com']['birthYear']?> (<a href="#"><?=$library['books'][3]['author']?></a>)</p>
```
Comment (Lehersun) [inline module_6/library.php:70]: If you are repeating code, something has written wrong. Always think about it. Here you should use one of commands for crawling an array like 'for', 'foreach' or 'while'

**Review by Lehersun — CHANGES_REQUESTED** [review]

File: module_6/library.php (hunk 2 above)
Comment (vladSHOKO) [reply module_6/library.php:70]: I know, but how can i use cycles in HTML code?

Comment (Lehersun) [reply module_6/library.php:70]:
```
<?php $elements = ['element 1', 'element 2', 'element 3']; ?>

<html>
    <body>
    <?php
    foreach ($elements as $element) { ?>
        <p> This is my element <?php echo $element ?> </p>
    <?php } ?>
    </body>
</html>

```

**Pull request closed without merging**